mllib/src/test/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifierSuite.scala

```diff
@@ -82,6 +83,49 @@ class MultilayerPerceptronClassifierSuite
     }
   }
 
+  test("strong dataset test") {
```

Comment:
Make the test title more descriptive so it is clear what it is testing.  E.g. "Predicted class probabilities: calibration on toy dataset"